yarn/datadog_checks/yarn/yarn.py

```diff
@@ -257,6 +258,16 @@ def _get_app_tags(self, app_json, app_tags):
                 self.log.error("Invalid value {} for application_tag".format(yarn_key))
         return tags
 
+    def collect_version(self, rm_address):
+        data = self._rest_request_to_json(rm_address, YARN_NODES_PATH, [])
```

Comment:
Given that this is the same request as line 275 (in `_yarn_node_metrics` should we do it as part of that to avoid an extra request? (Don't know how costly that request is)

Reply:
Yes I included version management in the code.